011-quarkus-panache-rest-flyway/src/main/resources/db/migration/V1.0.0__init.sql

```diff
@@ -1,4 +1,6 @@
-CREATE SEQUENCE hibernate_sequence START 1;
+-- TODO https://github.com/quarkusio/quarkus/issues/31451
```

Comment:
using more than one sequence seems to be correct strategy according to the https://github.com/quarkusio/quarkus/issues/31256, so I'm not sure about the TODO